data/yam/agama/auto/lib/storage.libsonnet

```diff
@@ -38,6 +38,20 @@ local lvm(encrypted=false, encryption='luks2') = {
       ],
     },
   ],
+  dasd: if dasd then {
```

Comment:
dasd is not related with lvm, I think it is better to create a function for dasd separately.  drives from existing config:
"storage": {
    "drives": [
      {
        "partitions": [
          {
            "search": "*",
            "delete": true
          },
          {
            "generate": "default"
          }
        ]
      }
    ]
  },

Reply:
I also missed where you read any lvm...

Reply:
From the ticket https://progress.opensuse.org/issues/183029:
"We could include this automation in existing scenario and update description saying that for s390x we add this difference, for example sles_lvm."

Reply:
sure, sorry, I should read better the ticket I write :) yes, this was a way to minimize the number of test suites, thanks.